doc/api/packages.md

```diff
@@ -126,21 +126,23 @@ There is the CommonJS module loader:
 * It is monkey patchable.
 * It supports [folders as modules][].
 * When resolving a specifier, if no exact match is found, it will try to add
-  extensions (`.js`, `.json`, and finally `.node`) and then attempt to resolve
+  extensions (`.js`, `.json`, `.node`, , and finally `.node`) and then attempt to resolve
   [folders as modules][].
 * It treats `.json` as JSON text files.
 * `.node` files are interpreted as compiled addon modules loaded with
   `process.dlopen()`.
 * It treats all files that lack `.json` or `.node` extensions as JavaScript
   text files.
-* It cannot be used to load ECMAScript modules (although it is possible to
-  [load ECMASCript modules from CommonJS modules][]). When used to load a
-  JavaScript text file that is not an ECMAScript module, it loads it as a
-  CommonJS module.
+* It can only be used to load synchronous ECMAScript modules (that contains
+  no top-level `await`) when `--experimental-require-module` is enabled. It
+  is also possible to [load ECMASCript modules from CommonJS modules][].
```

Comment:
```suggestion
* It can only be used to [load ECMASCript modules from CommonJS modules][] if
  the module graph is synchronous (that contains no top-level `await`) when
  `--experimental-require-module` is enabled.
```